shared/tracker/action.render.desktop.js

```diff
@@ -12,32 +12,32 @@ export default class ActionRender extends Component {
   props: ActionProps;
 
   render () {
-    const {username, loggedIn} = this.props
+    const {loggedIn} = this.props
 
     if (!loggedIn) {
-      return this.renderLoggedOut(styles, username)
```

Comment:
styles is module-level so we don't need it passed around. username isn't used cause its in the parent closure